### DISCUSSION TITLE
Fresh install, Cannot access test environment

### DISCUSSION THREAD

**Discussion by fenrir0wulf** [netbox-community/netbox#9932] [Q&A] [answered]
Hi all, completely new user, fresh install on Ubuntu 22.04. Followed and copy pasted every instruction thus far from https://docs.netbox.dev/en/stable/installation/
I've reached the end of Step 3, the Netbox install, with doing the runserver with the python script. However, even though everything has been exactly as described by the manual, when I try to access the web page, it's asking me to open an application called System Handler.
The ALLOWED_HOSTS has been configured with the wildcard.
![2022-08-05 10_52_30-Ubuntu - VMware Workstation](https://user-images.githubusercontent.com/33849693/183106907-e90419ec-12e3-488b-ae72-bcc4496c438a.png)

**Comment by cslingerland** [comment]
Try using a different browser. I believe it's a firefox issue.

**Comment by jeremystretch** [comment, marked as answer]
For some reason your browser seems to be interpreting `localhost` as the URL scheme rather than the hostname. Try adding `http://` in front of the URL.

> Try using a different browser. I believe it's a firefox issue.

FWIW I use Firefox primarily and have never run into this.

**Reply by fenrir0wulf** [reply]
This was it. I also use Firefox everyday in multiple environments and never encountered something like this.
